examples/robolectricExample/build.gradle

```diff
@@ -0,0 +1,51 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'de.undercouch:gradle-download-task:2.0.0'
+    }
+}
+
+apply plugin: 'de.undercouch.download'
+apply plugin: 'android-sdk-manager'
+apply plugin: 'com.android.application'
+apply plugin: 'android-command'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'realm-android'
+apply plugin: 'realm-robolectric'
+
+def currentVersion = file("${rootDir}/../version.txt").text.trim()
```

Comment:
Hmm odd. Because this line is in the main `build.gradle`:
```

allprojects {
    def currentVersion = file("${rootDir}/../version.txt").text.trim()
```

Perhaps @emanuelez or @zaki50 knows?